### DISCUSSION TITLE
Protein inference from casaNovo output

### DISCUSSION THREAD

**Discussion by 41ison** [Noble-Lab/casanovo#397] [Q&A] [answered]
Hi everyone,
The CasaNovo is a great tool for de novo search. Thanks for the development.
I'm a new user and everything seems to work well with the search.
But I'm curious, in the output file there are the columns: accession, unique, database, database_version, pre, post, start, end. They  come all NULL. Is there an option in config file or any command in the console that enables to load a fasta file to map the peptide sequences to proteins?

**Comment by bittremieux** [comment, marked as answer]
Casanovo performs de novo peptide sequencing, so by definition there is no database information available or needed. The empty columns are needed to follow the mzTab standard.

If you want to map the predicted peptides to a list of proteins, you'd have to implement a postprocessing script.
